### PR TITLE
Closes #49 Documentation: Updated "Get Started" page

### LINKED ISSUE #49: Documentation:  Updated "Get Started" page
### Please select a category the issue is focused on?

_No response_

### Let us know where something needs a refresh or put your idea here!

See [admiralpeds Get Started](https://pharmaverse.github.io/admiralpeds/articles/admiralpeds.html) as an example. Currently our page is empty.

https://pharmaverse.github.io/admiralpeds/articles/admiralpeds.html

### PR DESCRIPTION
**Pull request by nissinbo**
Closes #49 
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral family codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiral<ext> (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Review by AndersAskeland — CHANGES_REQUESTED** [review]
Thanks for the pull request @nissinbo ! 👍 

See my comments below. In addition, I think it would be great to add a "Derivations" section, where we quickly mention the ratio functions we have. See https://pharmaverse.github.io/admiralophtha/articles/admiralophtha.html for an example.

File: vignettes/admiralmetabolic.Rmd [hunk 1]
```diff
@@ -14,11 +14,43 @@ knitr::opts_chunk$set(
...
+
+`{admiralmetabolic}` also provides template R scripts as a starting point. They can be
+created by calling `use_ad_template()` from `{admiral}`, e.g.,
+
+```{r}
+library(admiral)
 ```
 
-Introduction to how to use the package extension...
+```r
```
[the remark this answers is not in the document]
Comment (nissinbo) [reply vignettes/admiralmetabolic.Rmd:46]: I agree, but looking at other admiral packages, they seem to separate these chunks for some reason.
https://pharmaverse.github.io/admiralonco/articles/admiralonco.html
https://pharmaverse.github.io/admiralpeds/articles/admiralpeds.html

File: vignettes/admiralmetabolic.Rmd [hunk 2]
```diff
@@ -14,11 +14,43 @@ knitr::opts_chunk$set(
...
+use_ad_template(
+  adam_name = "advs",
+  save_path = "./ad_advs.R",
+  package = "admiralmetabolic"
+)
```
[the remark this answers is not in the document]
Comment (manciniedoardo) [reply vignettes/admiralmetabolic.Rmd:51]: Yes, correct -`use_ad_template()` is an `{admiral}` function that should just work for us because we follow the `{admiral}` way of storing templates. So as long as you have `{admiralmetabolic}` installed (either from CRAN or as a dev) then you'll be fine. 

In fact I tested in a local session now:

image

File: vignettes/admiralmetabolic.Rmd (hunk 1 above)
[the remark this answers is not in the document]
Comment (AndersAskeland) [reply vignettes/admiralmetabolic.Rmd:46]: In my opinion, I don't see any reason to align here and would change it to one code chunk. Furthermore, remember that the `r` option in the code chunk should be inside "squiggly" brackets `{r}`

**Review by AndersAskeland — CHANGES_REQUESTED** [review]
I think we should change to one code chunk and add some text about adding issues on the github repository.

Besides that, it looks good to merge 👍

**Review by AndersAskeland — APPROVED** [review]
LGTM! Thank you for the work on this @nissinbo 👍
